apps/src/music/views/soundsPanel.module.scss

```diff
@@ -11,10 +11,15 @@
     display: flex;
     justify-content: space-between;
     margin-bottom: 5px;
+    gap: 8px;
 
     .segmentedButtons {
-      span {
-        font-size: 12px !important;
+      button {
+        padding: 7px 12px;
+
+        span {
+          font-size: 12px !important;
+        }
```

Comment:
Out of curiosity, does size `'s'` or `'xs'` provide similar styling without needing to manually override it here?

Reply:
Great call.  Updating to use `"s"` instead.